app/views/vzory/cookie-banner.njk

```diff
@@ -0,0 +1,697 @@
+                <li><a class="govuk-link" href="#prva-moznost" title="Možnosť 1 - stránka alebo služba používa iba nevyhnutné cookies">Možnosť 1 - stránka alebo služba používa iba nevyhnutné cookies</a></li>
+                <li><a class="govuk-link" href="#druha-moznost" title="Možnosť 2 - stránka alebo služba používa iné ako iba nevyhnutné cookies">Možnosť 2 - stránka alebo služba používa iné ako iba nevyhnutné cookies</a></li>
+                <li><a class="govuk-link" href="#tretia-moznost" title="Možnosť 3 - stránka alebo služba používa iné ako iba nevyhnutné súbory cookie, ale iba na strane klienta">Možnosť 3 - stránka alebo služba používa iné ako iba nevyhnutné súbory cookie, ale iba na strane klienta</a></li>
+                <li><a class="govuk-link" href="#co-ma-cookie-banner-obsahovat" title="Čo má cookie banner obsahovať">Čo má cookie banner obsahovať</a></li>
+                <li><a class="govuk-link" href="#priklad-pre-analyticke-cookies" title="Príklad pre službu s nevyhnutnými a analytickými súbormi cookie">Príklad pre službu s nevyhnutnými a analytickými súbormi cookie</a></li>
+                <li><a class="govuk-link" href="#priklad-pre-viac-druhov-cookies" title="Príklad pre službu alebo stránku využívajúcu viac ako jeden typ súborov cookie iných ako nevyhnutných">Príklad pre službu alebo stránku využívajúcu viac ako jeden typ súborov cookie iných ako nevyhnutných</a></li>
+                <li><a class="govuk-link" href="#doplnujuce-informacie" title="Doplňujúce informácie o fungovaní cookie banneru">Doplňujúce informácie o fungovaní cookie banneru</a></li>
+            </ul>
+
+
+            <h3 class="govuk-heading-m" id="kedy-pouzit-tento-vzor">Kedy použiť tento vzor</h3>
+            <p class="govuk-body">Cookie banner sa používa, ak služba nastavuje na zariadení používateľa akékoľvek súbory cookie.</p>
+            <p class="govuk-body">Majte na pamäti, že je potrebné:</p>
+            <ul class="govuk-list govuk-list--bullet">
+                <li>oboznámiť používateľov o súboroch cookie, ktoré služba nastavuje na ich zariadení,</li>
+                <li>umožniť používateľom prijať alebo odmietnuť akékoľvek súbory cookie, ktoré nie sú nevyhnutné pre fungovanie služby.</li>
+            </ul>
+
+            <h4 class="govuk-heading-s">Medzi nevyhnutné cookies nepatrí:</h3>
+            <ul class="govuk-list govuk-list-bullet">
+                <li>lokálne úložisko HTML5,</li>
+                <li>service workers,</li>
+                <li>akékoľvek ďalšia technológia, ktorá ukladá súbory do zariadenia používateľa.</li>
+            </ul>
+            <p class="govuk-body">Banner so súborom cookie a vzor stránky so súbormi cookie sú založené na prístupe k získaniu súhlasu s používaním súborov cookie na webových stránkach verejnej správy Slovenskej republiky.</p>
+            <p class="govuk-body">Táto stránka zobrazuje niekoľko možností použitia cookie banneru podľa typov súborov cookie, ktoré v službe používate.</p>
+
+
```

Comment:
len pre buducnost, zbytocne nemusi byt na viacerych miestach 2 volne riadky, staci jeden. Nepisal by som to ale je to na viacerych miestach